python/sbp/sbp2json.py

```diff
@@ -203,36 +193,25 @@ def sbp_main(args):
         read_offset += read_length
         buffer_remaining -= read_length
         while True:
-            if SBP_NO_JIT:
-                from construct.core import StreamError
-                bytes_available = read_offset - unconsumed_offset
-                b = buf[unconsumed_offset:(unconsumed_offset + bytes_available)]
-                if len(b) == 0:
-                    break
-                if b[0] != SBP_PREAMBLE:
-                    consumed = 1
-                else:
-                    try:
-                        m = msg_nojit.SBP.unpack(b)
-                        m = dispatch_nojit(m)
-                        dump(args, m)
-                        consumed = header_len + m.length + 2
-                    except StreamError:
-                        # Framing failed, find next preamble
-                        consumed = 1
-                    except ValueError as exc:
-                        # CRC error, skip malformed message and find next preamble
-                        consumed = header_len + exc.malformed_msg.length
+            from construct.core import StreamError
```

Comment:
Maybe break this import out if it is in a While True, I see that it was this way before though. Up to you.

Reply:
I kept it as is so the block is kept intact for `git blame`